config/mappings/nstf_mapping_parts/nstf_r2_scored_criteria.py

```diff
@@ -361,7 +361,7 @@
                 "name": "Targeted criteria",
                 "themes": [
                     {
-                        "id": "targeted_criteria",
+                        "id": "targeted_criteria-proposed",
```

Comment:
We're ok with a mix of _ and - compound id names? I suspect this may cause issues with any kind of concatenation interpretations :/

Reply:
good spot, that was just me being silly :S